openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/TagsAddRemove.spec.js

```diff
@@ -49,7 +51,27 @@ const removeTags = (tag, checkForParentEntity) => {
 
     cy.get('[data-testid="saveAssociatedTag"]').should('be.visible').click();
   } else {
-    cy.get(`[data-testid="remove-${tag}-tag"]`).should('be.visible').click();
+    if (isTable) {
+      cy.get(
+        '[data-testid="classification-tags-0"] [data-testid="edit-button"] '
+      )
+        .scrollIntoView()
+        .should('be.hidden')
+        .click();
```

Comment:
Why we are clicking on hidden? Instead fire hover event make it visible and then click